ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -232,18 +233,9 @@ echo "Launching 'wait-for bootstrap-complete' installation step....."
 # Let's use a proxy here as the internal net is not routable from the container running the installer.
 http_proxy="${proxy}" https_proxy="${proxy}" HTTP_PROXY="${proxy}" HTTPS_PROXY="${proxy}" \
   oinst agent wait-for bootstrap-complete 2>&1 &
-if ! wait $!; then
-  # TODO: gather logs??
-  echo "ERROR: Bootstrap failed. Aborting execution."
-  exit 1
-fi
 
```

Comment:
We still need that wait.

```suggestion
if ! wait $!; then
  echo "ERROR: Bootstrap failed. Aborting execution."
  exit 1
fi
```